interfaces/Matlab_Toolbox_Revamp/Class/Reactor.m

```diff
@@ -0,0 +1,334 @@
+            
+            checklib;
+            rho = calllib(ct, 'reactor_density', r.id);
+        end 
+
+        function mass = get.M(r)
+            % Get the mass of the reactor.
+            %
+            % return:
+            %    The mass of the reactor contents at the end of the
+            %    last call to 'advance' or 'step'. The mass is retrieved
+            %    from the solution vector. Unit: kg.
+            
+            checklib;
+            mass = calllib(ct, 'reactor_mass', r.id);
+        end
+        
+        function volume = get.V(r)
+            % Get the volume of the reactor.
+            %
+            % return:
+            %    The volume of the reactor contents at the end of the
+            %    last call to 'advance' or 'step'. Unit: m^3.
+            
+            checklib;
+            volume = calllib(ct, 'reactor_volume', r.id);
+        end          
+        
+        function enthalpy_mass = get.H(r)
+            % Get the mass specific enthalpy of the reactor.
+            %
+            % return:
+            %    The mass specific enthalpy of the reactor contents at the
+            %    end of the last call to 'advance' or 'step'. The enthalpy
+            %    is retrieved from the solution vector. Unit: J/kg. 
+            
+            checklib;
+            enthalpy_mass = calllib(ct, 'reactor_enthalpy_mass', r.id);
+        end        
+        
+        function intEnergy_mass = get.U(r)
+            % Get the mass specific internal energy of the reactor.
+            %
+            % return:
+            %    The mass specific internal energy of the reactor contents 
+            %    at the end of the last call to 'advance' or 'step'. The 
+            %    internal energy is retrieved from the solution vector. 
+            %    Unit: J/kg. 
+            
+            checklib;
+            intEnergy_mass = calllib(ct, 'reactor_intEnergy_mass', r.id);
+        end
+        
+        function yi = massFraction(r, species)
+            % Get the mass fraction of a species.
+            %
+            % parameter species:
+            %    String or one-based integer id of the species.
+            
+            checklib;
+            
+            if ischar(species)
+                k = r.contents.speciesIndex(species) - 1;
+            else k = species - 1;
+            end
+            
+            yi = calllib(ct, 'reactor_massFraction', r.id, k);
+        end
+        
+        function massFractions = get.Y(r)
+            % Get the mass fractions of the reactor.
+            %
+            % return:
+            %    The mass fractions of the reactor contents at the end of 
+            %    the last call to 'advance' or 'step'.
+            
+            checklib;
+            
+            nsp = r.contents.nSpecies;
+            y = zeros(1, nsp);
+            for i = 1:nsp
+                y(i) = r.massFraction(i);
+            end
+        end        
```

Comment:
Do getters need to `return` in the new Matlab class style? I don't see how Matlab knows that `y` is the "output" since it's not in the function definition line anymore.

Reply:
That was a mistake. Fixed.